### ISSUE TITLE
NTR: heterochromatin disassembly

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#26716]
Please provide as much information as you can: 

There may be a reason why we do not have this term, but I can't remember what it is.

I need it to annotate epe1
https://www.pombase.org/reference/PMID:31206516

Collectively, the results show that Epe1 has two distinct functions: the protection of euchromatic regions from stochastic de novo ectopic heterochromatin formation via a mechanism involving its N-terminal transcriptional activation (NTA) domain, and incomplete **disruption of already-established ectopic heterochromatin via its JmjC domain.** Thus, Epe1 could be a key regulator in the formation of individual-specific H3K9me landscapes. 

Is there another term I should use? @pgaudet @colinlog ?

* **Suggested term label:**
NTR heterochromatin disassembly

* **Definition (free text)**
An epigenetic mechanism in which chromatin which is compacted into heterochromatin is disassembled. 

* **Reference, in format PMID:#######**
 
PMID:31206516

* **Gene product name and ID to be annotated to this term**

epe1

* **Parent term(s)**

I don't know what the parent would be? GO:0070828 heterochromatin organization
doesn't work because that is
Any process that results in the specification, formation or maintenance of the physical structure of eukaryotic heterochromatin, a compact and highly condensed form of chromatin.

* **Any other information**

**Comment by cmungall** [comment]
We should document general expectations of disassembly terms here

https://github.com/geneontology/go-ontology/blob/master/docs/patterns/cc_disassembly.md

X disassembly is typically under X assembly

<img width="427" alt="image" src="https://github.com/geneontology/go-ontology/assets/50745/e07a8cf6-ae07-4d6d-aaf7-38192ca1a21f">

**Comment by ValWood** [comment]
Do you mean under organization?
That's true, but this particular organization term does not seem to include disassembly (maybe because the term did not exist). I  first wanted to check if there is some reason why it does not exist in the current arrangement.

**Comment by ValWood** [comment]
This is my problem:

I have used negative regulation of heterochromatin assembly for  Eri1 
this prevents the formation of heterochromatin
https://www.pombase.org/reference/PMID:16797182
I'm OK with this one.

However, Epe1 disrupts/disassembles existing heterochromatin,
this is probably a better paper to demonstrate this:
PMID:16762840 Swi6/HP1 recruits a JmjC domain protein to facilitate transcription of heterochromatic repeats.
Authors	Zofall M, Grewal SI

Epe1 appears to be involved in the destabilisation of existing chromatin to allow polymerase access in order to transcribe repeat elements:

- These results suggest either that Epe1 binds specifically to the boundary elements, thus pre- venting the spread of heterochromatin, or that Epe1 weakens heterochromatin spreading by directly interfering with the stability of heterochromatin complexes. To address these possibilities, we explored Epe1 distribution by using chromatin immunoprecipitation (ChIP) analyses and fluorescent microscopy. 

- We thus conclude that Epe1 facilitates transcription of repeats within the context of heterochromatin by destabilizing repressive chromatin complexes.

- Remarkably, loss of Epe1 resulted in a diminished access of Pol II to cenH in Dclr3 cells, as indicated by substantial reduction in Pol II levels in Depe1 Dclr3 double mutant as compared to Dclr3 single mutant (Figure 5E). These results suggest that Epe1 has an important role in promoting access of transcriptional machinery to heterochromatic sequences, thereby facilitating transcription of repeat elements. Therefore, the balance of activities between Clr3 and Epe1, both of which are recruited by Swi6, seems critical in determining the transcriptional state of repeat ele- ments.

- The Y307 mutation abolished the ability of Epe1 to destabilize heterochromatic silencing (Figure 6A). 
(deMethylase)

- Moreover, we were unable to detect histone demethylase activity for Epe1 (data not shown). Thus, although the JmjC domain of Epe1 is essential for modulating heterochromatin, the mechanism utilized might be distinct from other JmjC domain proteins known to possess demethylase activity. Indeed, loss of Epe1 resulted in increased Clr4 HMTase occu- pancy and H3K9 methylation levels at the ssm4 locus (Figures 6F and 6G), as well as an increase in Clr3 levels at heterochromatic loci (our unpublished data). Therefore, it is possible that JmjC domain of Epe1 affects het- erochromatin stability by directly interfering with localization of other chromatin-modifying factors.

I just feel that these are pathways operating  in different context and on different heterochromatin substrates and so it seems strange to lump them together? Maybe I'm overthinking it?

**Comment by pgaudet** [comment]
It seems like that second role is wither 'GO:0045815 transcriptional initiation-coupled chromatin remodeling' or 'GO:0140673 transcriptional elongation-coupled chromatin remodeling'

**Comment by colinlog** [comment]
Hello Val, here my insights which I hope can help you. Firstly I discuss Epe1. Then the concept disassembly


I read the following papers: 
https://genesdev.cshlp.org/content/27/17/1886 --> Epe1 and bromodomain recruitment 2013

https://doi.org/10.7554/eLife.90525 --> is a 2024 paper that further confirms the 'anti-silencing' activity of Epe1 and also shows that it is present at many if not all heterochromatin loci.

I conclude that the BP is still heterochromatin formation, but Epe1 is involved in negative regulation of heterochromatin formation. This appears to be through promotion of the BP: 'Transcription intiation-coupled chromatin remodelling' since the 2013 paper indicates a positive involvment in transcription at heterochromatin boundaries.

Furthermore,
https://pubmed.ncbi.nlm.nih.gov/35171902/ --> shows that Epe1 is regulated by the cAMP pathway.

and

https://pubmed.ncbi.nlm.nih.gov/35171902/ --> states "Furthermore, H2A.Z suppresses Epe1-mediated transcriptional activation, which is required for subtelomeric gene repression."

Therefore, perhaps the best annotations you could make for the 2006 Epe1 paper, would be (i) to annotate Epe1 to the BP GO:0033696 heterochromatin boundary formation and (ii) To annotate Epe1 to GO:0045815 'Transcription intiation-coupled chromatin remodelling'.  I understand you may believe this last is too specific (or too vague) a term. In that case, Epe1 could be annotated to the process: GO:0141137 positive regulation of gene expression, epigenetic.

Overall: Just recently it has been shown that heterochromatin,though more compact, is not less accessible than euchromatin. Hence, it is all about what can and what cannot happen in these two opposing types of chromatin, rather than to think in terms of 'closed' and 'open'. What can and cannot is often dictated by the H3K9 and H3K27 PTMs methylation or acetylation. If methylated, the lysine cannot tbe acetylated, and vice versa.  

Epe1, by (conditionally?) recruiting the Bromodomain protein Bdf2 likely allows RNA polymerase II to bind to bona-fide gene promoters and allow the balance to be tipped from heterochromatin towards euchromatin. Which is why I believe you can annotate it to GO:0045815 'Transcription intiation-coupled chromatin remodelling'

Epe1 appears to play a negative role in a fungi (+ plant)-specific pathway that requires RNA-template-dependent RNA polymerases to amplify rare transcripts originating in heterochromatin that participate in the heterochromatin formation process. Epe1 appears to be required to form boundaries for heterochromatin as well as to conditionally (heat shock, high glucose) drive the transcription process that will transform a heterochromatin region in euchromatin.  This is indirectly disassembly of heterochromatin. But, directly it appears to be RNA polymerase II transcription promotion.

**Comment by ValWood** [comment]
Thanks @colinlog  this is really, really helpful

**Comment by pgaudet** [comment]
@ValWood 
Can we close this ? Maybe this can be a synonym of another term ? Perhaps on [negative regulation of heterochromatin formation](http://purl.obolibrary.org/obo/GO_0031452)?

**Comment by ValWood** [comment]
yes, we can annotate to negative regulation of heterochromatin formation.

CC @PCarme
